src/google.js

```diff
@@ -95,5 +95,21 @@ module.exports = (() => ({
     } catch (err) {
       throw new Error(`Unable to create contact ${err.message}`)
     }
+  },
+  updateContact: function(person) {
+    const peopleAPI = google.people({
+      version: 'v1',
+      auth: this.oAuth2Client
+    })
+
+    try {
+      return peopleAPI.people.updateContact({
+        resourceName: person.resourceName,
+        requestBody: person
+        // TODO: updatePersonFields
```

Comment:
Tu peux passer un objet complet et google se débrouille tout seul ? Ou il faut passer que les champs à modifier ?

Reply:
the API wants a field mask : https://github.com/konnectors/cozy-konnector-google/commit/58196208689570337209272832995b5814e81e6e